pkg/repository/container_redis.go

```diff
@@ -201,16 +215,9 @@ func (cr *ContainerRedisRepository) GetWorkerAddress(containerId string) (string
 	}
 }
 
-// GetActiveContainersByStubId gets active containers by stub ID.
-// stubId is the external ID.
-func (cr *ContainerRedisRepository) GetActiveContainersByStubId(stubId string) ([]types.ContainerState, error) {
-	indexKey := common.RedisKeys.SchedulerContainerIndex(stubId)
-	keys, err := cr.rdb.SMembers(context.TODO(), indexKey).Result()
-	if err != nil {
-		return nil, fmt.Errorf("failed to retrieve container state keys: %v", err)
-	}
+func (cr *ContainerRedisRepository) listContainerKeysByIndexKeys(indexKey string, keys []string) ([]types.ContainerState, error) {
```

Comment:
rename -> listContainerStateByIndex or something like that and add a comment about what the func does

Reply:
updated